### PR TITLE
docs: improve bareos-webui documentation

### PR DESCRIPTION
**Pull request by bruno-at-bareos**
This PR aims to merge both chapter bareos-webui and bareos-webui-install(update).
With this move, end-user should find all webui related information in one place.

- merge BareosWebuiInstall.rst into BareosWebui.rst
- add instruction for AppArmor protected system
- Fix Bareos links redirection issues in WhatIsBareos.rst

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
- [x] Check backport line

### CODE REVIEW

**Review by joergsteffens — CHANGES_REQUESTED** [review]
Looks good. I just found one typo. The other suggestions just remove information about outdated versions.
Maybe you can additonally remove section https://download.bareos.org/experimental/PR-1366/BareosMainReference/TasksAndConcepts/TransportEncryption.html#configuration-example-for-bareos-17-2 ?

And I'm unsure where the best place for this chapter will be. An alternative would be after the tuturial chapter.

**Comment by joergsteffens** [comment]
The pr-tool complains:
```
 ✗  bareos-check-sources --since=47d9cdbd98cb28a0d6cabf2391b82884c6a35fd0 reported:
	Plugin 'trailing newlines' would modify 'docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst'
```
Also I think the commit messages should start with "docs:".

File: docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst [hunk 1]
```diff
@@ -4,6 +4,685 @@
...
+The configuration parameter has been introduced because in case of larger backup jobs with lots of files
+which are not already present in the Bvfs cache you could run into timeouts while trying to load the filetree
+in the restore module of the |webui|.
+
+If you have trouble with running into timeouts while loading the tree you can adjust the parameter ``filetree_refresh_timeout``
+to your needs. Keep in mind to set the timeout in your Apache or Nginx configuration accordingly to the setting
+in your :file:`configuration.ini`.
+
+In general we highly recommend updating the Bvfs cache frequently. Please see :ref:`section-updating-bvfs-cache-frequently` for further details on how to accomplish this.
+
```
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst:230]: Thanks I think so also.
@fbergkemper any through on that.

**Comment by bruno-at-bareos** [comment]
I've fixed the new lines and also changed the PR subject to start with docs:
@joergsteffens shall I do a squash of commits before you go for green light?

**Comment by joergsteffens** [comment]
Yes, please. What about my other two remarks from https://github.com/bareos/bareos/pull/1366#pullrequestreview-1287379777
Do you want to keep it the way it is?

**Comment by bruno-at-bareos** [comment]
> Yes, please. What about my other two remarks from [#1366 (review)](https://github.com/bareos/bareos/pull/1366#pullrequestreview-1287379777) Do you want to keep it the way it is?

No, thanks for the reminders. At least I will remove the obsolete 17 part.

File: docs/manuals/source/TasksAndConcepts/TransportEncryption.rst [hunk 2]
```diff
@@ -418,50 +418,6 @@ Please check the following configuration examples.  A complete table of the dire
...
-     TLS Require = no
-     TLS Verify Peer = no
-     TLS CA Certificate File = /etc/bareos/tls/ca.crt
-     TLS Certificate = /etc/bareos/tls/server.crt
-     TLS Key = /etc/bareos/tls/server.pem
-   }
-
-
-
 Configuration example for Bareos 18.2
```
Comment (joergsteffens) [inline docs/manuals/source/TasksAndConcepts/TransportEncryption.rst:421]:
```suggestion
Configuration example
```

Comment (joergsteffens) [reply docs/manuals/source/TasksAndConcepts/TransportEncryption.rst:421]: And could you please also prepend the git commit message with "docs:" ? Or also squash them together.

Comment (bruno-at-bareos) [reply docs/manuals/source/TasksAndConcepts/TransportEncryption.rst]: all done in last push.

**Review by joergsteffens — APPROVED** [review]

**Pull request merged**